pdns/lua-record.cc

```diff
@@ -945,6 +952,27 @@ static void setupLuaRecords()
         });
 
     });
+  lua.writeFunction("countryCode", []() {
+      string res = getGeo(s_lua_record_ctx->bestwho.toString(), GeoIPInterface::Country2);
+      if ( res == "unknown" ) {
+       res = "--";
```

Comment:
Are *all* these strings stored in the text segment?

Or, another way, is this the only string that doesn't need to be deallocated?

Reply:
The string handling is similar to the other, existing, methods ("asnum" for example). 
The "unknown" bit is a default response from getGeo in the case when there is no matching country, since it is all two-letter ISO I decided to use "--" instead.